com.unity.visualeffectgraph/Editor/Types/VFXPropertyAttribute.cs

```diff
@@ -175,6 +179,11 @@ public VFXExpression ApplyToExpressionGraph(VFXExpression exp)
                     var enumAttribute = (EnumAttribute)attribute;
                     exp = new VFXExpressionMin(exp, VFXValue.Constant((uint)enumAttribute.values.Length - 1));
                 }
+                else if (attribute is MinMaxAttribute)
```

Comment:
If the C# version is at least 7 you can take advantage of new pattern matchin:
`if (attribute is MinMaxAttribute minMaxAttribute)`